packages/core/src/render3/util/global_utils.ts

```diff
@@ -45,51 +69,34 @@ export function publishDefaultGlobalUtils() {
     _published = true;
 
     setupFrameworkInjectorProfiler();
-    publishGlobalUtil('ɵgetDependenciesFromInjectable', getDependenciesFromInjectable);
-    publishGlobalUtil('ɵgetInjectorProviders', getInjectorProviders);
-    publishGlobalUtil('ɵgetInjectorResolutionPath', getInjectorResolutionPath);
-    publishGlobalUtil('ɵgetInjectorMetadata', getInjectorMetadata);
-    /**
-     * Warning: this function is *INTERNAL* and should not be relied upon in application's code.
-     * The contract of the function might be changed in any release and/or the function can be
-     * removed completely.
-     */
-    publishGlobalUtil('ɵsetProfiler', setProfiler);
-    publishGlobalUtil('getDirectiveMetadata', getDirectiveMetadata);
-    publishGlobalUtil('getComponent', getComponent);
-    publishGlobalUtil('getContext', getContext);
-    publishGlobalUtil('getListeners', getListeners);
-    publishGlobalUtil('getOwningComponent', getOwningComponent);
-    publishGlobalUtil('getHostElement', getHostElement);
-    publishGlobalUtil('getInjector', getInjector);
-    publishGlobalUtil('getRootComponents', getRootComponents);
-    publishGlobalUtil('getDirectives', getDirectives);
-    publishGlobalUtil('applyChanges', applyChanges);
+    for (const [methodName, method] of Object.entries(globalUtilsFunctions)) {
+      publishGlobalUtil(methodName as GlobalUtilsFunctions, method);
+    }
   }
 }
 
-export declare type GlobalDevModeContainer = {
-  [GLOBAL_PUBLISH_EXPANDO_KEY]: {[fnName: string]: Function};
+/**
+ * Default debug tools available under `window.ng`.
+ */
+export type GlobalDevModeContainer = {
```

Comment:
nit: I'm wondering if we could name it like this, which looks a bit more specific:
```suggestion
export type GlobalDevModeUtils = {
```